peachjam/templates/peachjam/terms_of_use.html

```diff
@@ -0,0 +1,33 @@
+{% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
+
+{% block title %} {% trans 'Terms of Use' %}{% endblock %}
+
+{% block page-content %}
+  <section class="py-5" data-component="TermsOfUse">
+    <div class="container">
+      <div class="row mb-4">
+        <div class="col-12 col-md-3"></div>
+        <div class="col-12 col-md-9">
+          <h1>
+            {% block page-title %} {% trans 'Terms of Use' %}{% endblock %}
+          </h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12 col-md-3">
+          <div class="list-group mb-4">
+            <div class="list-group-item bg-light">
+              <strong>{% trans 'Navigation' %}</strong>
+            </div>
+            <div class="list-group-item" data-table-of-contents>
```

Comment:
Minor: a class name would work equally well here and is arguably exactly what classes are for, rather than a custom attribute.

Reply:
done